ShenkinStore/Models/ShoppingCart.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShenkinStore.Models
+{
+    public class ShoppingCart
```

Comment:
Remove unnecessary comments and break lines from here

Reply:
done